src/lib/rest/StringFilter.cpp

```diff
@@ -441,6 +442,15 @@ bool StringFilterItem::valueGet
     *stringP = s;
   }
 
+  if (*valueTypeP == SfvtString)  // FIXME PR: perhaps not calling forbiddenChars if pattern matching ...
```

Comment:
Discussed over skype and decicion made to send OP down to StringFilterItem::valueGet to avoid doing forbiddenChars for pattern matching